app/views/admin/health/patient_referrals/_reject.haml

```diff
@@ -9,17 +9,18 @@
 .patient-referral__reject
   = simple_form_for pr, url: admin_health_patient_referral_reject_path(pr), namespace: dropdown_id, remote: true do |f|
     = f.input :rejected_reason, as: :hidden
-  - if pr.stop_reason_description
-    .mb-4
-      %strong #{@sender.receiver_name} Disenrollment Reason
-      %br
-      = pr.stop_reason_description
-  .dropdown
-    = link_to '#', id: dropdown_id, data: {toggle: 'dropdown'}, aria: {haspopup: 'true', expanded: 'false'}, class: 'btn btn-secondary btn-sm' do
-      = label
-      %span.caret
-    %ul.dropdown-menu{aria: {labelledby: dropdown_id}}
-      - Health::PatientReferral.rejected_reasons.each do |reason|
-        - unless reason[0] == pr.rejected_reason
-          %li
-            = link_to Health::PatientReferral.display_rejected_reason(reason[0]), '#', data: {behavior: 'reject patient referral', rejected_reason: reason[0]}
+    - if pr.stop_reason_description
+      .mb-4
+        %strong #{@sender.receiver_name} Disenrollment Reason
+        %br
+        = pr.stop_reason_description
+    .dropdown
+      = link_to '#', id: dropdown_id, data: {toggle: 'dropdown'}, aria: {haspopup: 'true', expanded: 'false'}, class: 'btn btn-secondary btn-sm' do
+        = label
+        %span.caret
+      %ul.dropdown-menu{aria: {labelledby: dropdown_id}}
+        - Health::PatientReferral.rejected_reasons.each do |reason|
+          - unless reason[0] == pr.rejected_reason
+            %li
+              = link_to Health::PatientReferral.display_rejected_reason(reason[0]), '#', data: {behavior: 'reject patient referral', rejected_reason: reason[0]}
+    = f.input :disenrollment_date, as: :date_picker,  label: 'Disenrollment date:', input_html: { value:pr.disenrollment_date || Date.current.end_of_month }
```

Comment:
shouldn't this be `value: pr.pending_disenrollment_date || Date.current.end_of_month`?

Reply:
Also, maybe we should move that to `- value = pr.pending_disenrollment_date || Date.current.end_of_month` and then just use `value: value`